grails-app/services/org/pih/warehouse/core/LocalizationService.groovy

```diff
@@ -65,7 +65,9 @@ class LocalizationService {
 	 */
 	Locale getCurrentLocale() {
 		// fetch the locale of the current user; if there isn't one, use the default locale
-		return (RequestContextHolder.currentRequestAttributes().getSession().user?.locale ?: new Locale(grailsApplication.config.openboxes.locale.defaultLocale))
+		return RequestContextHolder.currentRequestAttributes().getSession().user?.locale?.language != "null" ?

```

Comment:
@DominikaHatala Can you explain why this was necessary?

Reply:
Nevermind, it seems like some users have a "null" locale setting. I'd prefer to keep the code as-is and just fix the data issue by setting the locale to something not equal to "null".